src/main/webapp/styles/personalised-recommendations.css

```diff
@@ -0,0 +1,91 @@
+#instructions {
+  color: black;
+}
+
+/* Wrapper Containing Preference Form and Recommendations */
+.wrapper {
+  display: flex;
+  width: 50%;
```

Comment:
I am also not sure about this, but just an idea. since we have this attribute in .preference-container, can we delete this?Is it possible to move the attribute to preferences-container?

Reply:
Yes, the wrapper container will be used for containing the recommendations too, but I can change this if you think we don't need it?